versions.gradle

```diff
@@ -0,0 +1,154 @@
+/**
+ * Shared file between builds so that they can all use the same dependencies and
+ * maven repositories.
+ **/
+ext.deps = [:]
+def versions = [:]
+versions.arch_core = "1.1.0"
```

Comment:
The project doesn't use all dependencies listed here, please make sure you shrink those down to only used dependencies.

Reply:
Sure, I'll do the clean up